engine/src/main/java/org/terasology/engine/monitoring/chunk/ChunkMonitor.java

```diff
@@ -61,17 +61,17 @@ public static void fireChunkCreated(Chunk chunk) {
 
     public static void fireChunkDisposed(Chunk chunk) {
         Preconditions.checkNotNull(chunk, "The parameter 'chunk' must not be null");
-        post(new ChunkMonitorEvent.Disposed(chunk.getPosition(new Vector3i())));
+        post(new ChunkMonitorEvent.Disposed(new Vector3i(chunk.getPosition())));
```

Comment:
Is this conversion an artifact of `ChunkMonitorEvent.Disposed` requiring a mutable `Vector3i` instead of a `Vector3ic` as returned by `Chunk`? If so, is there any need for it to be a mutable vector, or could we adjust the interface to just work with `Vector3ic`?